github-actions/review-release/action.yml

```diff
@@ -0,0 +1,49 @@
+name: 'Review Release'
+description: 'Determines whether a given app release is automatically approved for Splunkbase upload or if it requires manual approval.'
+inputs:
+  github_token:
+    description: 'Token used to authenticate with the GitHub REST API'
+    required: true
+outputs:
+  manual_review_required:
+    description: "true/false depending on if the the release requires manual review"
+    value: ${{ steps.review_release.outputs.manual_review_required }}
+runs:
+  using: 'composite'
+  steps:
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.9'
+    - name: 'Check PIP Cache'
+      uses: actions/cache@v2
+      id: cache
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-approve-release
```

Comment:
minor: the setup-python v3 action supports a `cache` key directly instead of needing two separate actions